packages/core/src/odata-common/filter/filter-lambda-expression.ts

```diff
@@ -13,10 +12,7 @@ export type FilterLambdaOperator = 'any' | 'all';
 /**
  * hidden
  */
-export class FilterLambdaExpression<
-  EntityT extends Entity,
-  FieldT extends FieldType
-> {
+export class FilterLambdaExpression<EntityT extends Entity> {
```

Comment:
I don't see an issue here, but @jjtang1985 can you double check?

Reply:
I guess it should be fine since it is `hidden`?

Reply:
Although it is exposed as part of Filterable...